### PR TITLE
refactored PR tests to take a common folder name parameter

### PR DESCRIPTION
**Pull request by ambrosejcarr**
This makes some minor changes to the already-existing documentation for the PR test to remove some test-specific information from the Jenkins job, making it easier to create a new jenkins job. 

The format to make a new jenkins job is now just to copy the existing job, and change the `PIPELINE_FOLDER_NAME=<optimus|smartseq2_single_sample>` parameter to point to the new testing directory. 

... and for some reason, the PR carries over some changes to the SS2 readme.

### CODE REVIEW

**Comment by ambrosejcarr** [comment]
@dshiga @jacmrob When I run this test, the (failed) integration test is declared as "blocking" the Jenkins SS2 Tests. Thus, the SS2 test isn't running. Can you guys comment on what's going on?

**Comment by dshiga** [comment]
I'm not sure why it's set up to block the ss2 tests, and don't think we want to keep it that way, but have you identified the reason for the integration test failure?

**Comment by ambrosejcarr** [comment]
It definitely shouldn't be anything I've done; the only changes here are to documentation and the name of the PR test wdls. 

So long as the integration test doesn't run the PR tests there should be no problems that result from my changes.

**Comment by dshiga** [comment]
@samanehsan could you please work with @ambrosejcarr to figure out the source of the integration test failure?

**Comment by ambrosejcarr** [comment]
jenkins retest

**Comment by ambrosejcarr** [comment]
jenkins retest

**Comment by ambrosejcarr** [comment]
@dshiga @samanehsan The SS2 test was created with a weird configuration. I've changed it to replicate the configuration for Optimus and both PR tests appear to be working now.

**Comment by dshiga** [comment]
that's good, thanks @ambrosejcarr! @samanehsan has a fix in review for the integration test: https://github.com/HumanCellAtlas/secondary-analysis/pull/11

File: test/README.md [hunk 1]
```diff
@@ -28,9 +28,10 @@ More complex tests may require different inputs, but the expected values should
 The jenkins job executes as follows: 
 ```bash
 VAULT_TOKEN=$(cat /etc/vault-token-dsde)
-INPUTS_JSON="https://raw.githubusercontent.com/HumanCellAtlas/skylab/master/test/optimus/pr/test_inputs.json"
-WDL_FILE="/working/test/optimus/pr/TestOptimusPR.wdl"
-DEPENDENCIES_JSON="/working/test/optimus/pr/dependencies.json"
+PIPELINE_FOLDER_NAME=<optimus|smartseq2_single_sample>
+INPUTS_JSON="https://raw.githubusercontent.com/HumanCellAtlas/skylab/master/test/${PIPELINE_FOLDER_NAME}/pr/test_inputs.json"
+WDL_FILE="/working/test/${PIPELINE_FOLDER_NAME}/pr/Test${PIPELINE_FOLDER_NAME}PR.wdl"
```
Comment (samanehsan) [inline test/README.md]: This should say `test_${PIPELINE_FOLDER_NAME}_PR` instead of `Test${PIPELINE_FOLDER_NAME}PR`

File: test/README.md [hunk 2]
```diff
@@ -28,9 +28,10 @@ More complex tests may require different inputs, but the expected values should
...
+DEPENDENCIES_JSON="/working/test/${PIPELINE_FOLDER_NAME}/pr/dependencies.json"
```
Comment (samanehsan) [inline test/README.md]: Are we still using a dependencies.json file for the tests?

**Comment by samanehsan** [comment]
Just a couple of comments/questions otherwise this is good to merge!

File: test/README.md (hunk 2 above)
Comment (ambrosejcarr) [reply test/README.md]: We are. Do you have a suggestion to remove them?

Comment (samanehsan) [reply test/README.md]: Oh, I'm thinking of how lira isn't using the dependencies.json file to run the adapter wdls anymore. Nevermind!

**Pull request merged**